build.gradle

```diff
@@ -601,6 +601,7 @@ jlink {
         requires 'jdk.security.jgss'
         requires 'jdk.xml.dom'
         requires 'com.google.gson'
+        requires 'org.jsoup'
```

Comment:
Interesting. `module-info.java` has this info: https://github.com/JabRef/jabref/blob/16a526f089a8f9e0a25d4a40faaba32cc83e6e03/src/main/java/module-info.java#L82

Reply:
Yes, but int the end this is all merged with a module and the plugin sometimes cannot determine the whole modules correctly.  Failing then at runtime

Reply:
Flexmark depends on jsoup and jsoup is already modularized and would not be included in the merged module and therefore cannot be found I guess